packages/react-vapor/src/components/button/examples/ButtonExamples.tsx

```diff
@@ -3,6 +3,7 @@ import {Svg} from '../../svg/Svg';
 import {Button} from '../Button';
 
 export class ButtonExamples extends React.Component<any, any> {
+    static description = 'Buttons are used to communicate and initialize actions when the user clicks them.';
     render() {
         return (

```

Comment:
While at it, it would be nice to add print delimiters to scope only the source code that is relevant to the example (i.e. what is inside the return parenthesis in the `render` method).

start delimiter: `// start-print`
stop delimiter: `// stop-print`